UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/HealthCheck/EventHubLoggingHealthClient.cs

```diff
@@ -31,7 +34,17 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
 
             try
             {
-                await eventHubClient.SendAsync(new EventData(Encoding.UTF8.GetBytes(EventIds.EventHubLoggingEventDataForHealthCheck.ToEventId() + " of Event Hub")));
+                var logEntry = new LogEntry
+                {
+                    Timestamp = DateTime.UtcNow,
+                    Level = LogLevel.Information.ToString(),
+                    MessageTemplate = "Event Hub Logging Event Data For Health Check",
+                    EventId = EventIds.EventHubLoggingEventDataForHealthCheck.ToEventId()
+                };
```

Comment:
As these events will appear in Kibana, we should also be adding the default logging properties:
* System
* Service
* NodeName
* Environment.

These are available from the `eventHubLoggingConfiguration`. See also `UKHO.ExchangeSetService.API\UKHO.ExchangeSetService.API\Startup.cs`

Reply:
Thanks, will revise

Reply:
I think it will be as simple as:

```suggestion
                    LogProperties ={
                        {"_System", eventHubLoggingConfiguration.Value.System},
                        {"_Service", eventHubLoggingConfiguration.Value.Service},
                        {"_NodeName", eventHubLoggingConfiguration.Value.NodeName},
                        {"_Environment", eventHubLoggingConfiguration.Value.Environment}
                    }
                };
```